src/webgl/p5.Geometry.js

```diff
@@ -88,6 +88,10 @@ p5.Geometry = class Geometry {
     this.dirtyFlags = {};
   }
 
+  clearColors() {
```

Comment:
Do you think you can add a doc comment around this one? It would be nice to add your example in the video in your PR description. Maybe we can add two examples: one showing the behaviour without `clearColors()`, and one showing the behaviour with `clearColors()`?

Here's an example of a doc comment with an example sketch on another class. Adding one here would look similar: https://github.com/processing/p5.js/blob/716554fd780f28390dbcfdcf4ac85304750d8211/src/webgl/p5.Framebuffer.js#L750-L794

Reply:
Okay, I will add a doc comment around this one.